readme.md

```diff
@@ -113,6 +113,22 @@ our community aspires to be a respectful place both during online and in-person
 - [Submit a Pull Request](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request) to the main branch.
 - After any questions or changes have been resolved, your contribution would be merged in!
 
+## Creating a .env file to use your own GitHub API token
```

Comment:
Lets move this to the`### Code changes` section after assets.

Also, change the heading to just creating an env file - scalable in future, in case we decide to add more vars.

Reply:
That makes sense, great to learn how the concept of scalability can be applied to something as simple as documentation.